jsapp/js/components/processing/analysis/responseForms/responseWrapper.component.tsx

```diff
@@ -97,6 +99,12 @@ export default function ResponseWrapper(props: ResponseWrapperProps) {
     }
   }
 
+  function handleClear() {
+    if (typeof props.clearable === 'function') {
+      props.clearable()
+    }
+  }
```

Comment:
consider to add `event.stopPropagation()`

Reply:
Not sure why I'd need this? No other buttons have this on their listeners, and the parent element does not have any event listeners to propagate to